corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -8,6 +8,27 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         toggles = hqImport("hqwebapp/js/toggles"),
         utils = hqImport("cloudcare/js/formplayer/utils/utils");
 
+    const locationIcon = L.divIcon({
+        className: 'custom-div-icon',
+        html: "<div style='background-color:#0b3fc3;' class='marker-pin'></div>",
+        iconSize: [30, 42],
+        iconAnchor: [15, 42]
+    });
+
+    const selectedLocationIcon = L.divIcon({
+        className: 'custom-div-icon',
+        html: "<div style='background-color:#c39b0b;' class='marker-pin'></div>",
+        iconSize: [30, 42],
+        iconAnchor: [15, 42]
+    });
+
+    const homeLocationIcon = L.divIcon({
+        className: 'custom-div-icon',
+        html: "<div style='background-color:#c30b82;' class='marker-pin'></div>",
+        iconSize: [30, 42],
+        iconAnchor: [15, 42]
+    });
```

Comment:
I think the dimensions should be local variables instead of hard coded, but could go either way on moving them into the const file. They (either the dimensions or the icons themselves) could go into map_utils, but I don't think they _need_ to, since they're only used in this file.

The styles are bugging me, though, could those be classes? I'm pretty dogmatic about avoiding inline styles, if you haven't already noticed, color codes in particular - we've done look & feel overhauls in the past, and having to search all the html & js for random color codes is pretty miserable.

Reply:
new pin version does not use inline style anymore